R/recipe_run.R

```diff
@@ -69,11 +67,11 @@ orderly_run <- function(name, parameters = NULL, envir = NULL,
 
 ##' @export
 ##' @rdname orderly_run
-orderly_data <- function(name, parameters = NULL, envir = list(),
+orderly_data <- function(name, parameters = NULL, envir = NULL,
```

Comment:
Using a list here was not used internally and I don't think anyone else needed it so I pulled it out. Doing so simplifies things a little